### PR TITLE
Add more pages to navigation for webKnossos docs

### PR DESCRIPTION
**Pull request by hotzenklotz**
### Description:
- I added and re-arranged the pages in the webKnossos docs navbar

### Issues:
- Related to webKnossos docs update PR: https://github.com/scalableminds/webknossos/pull/5995

### Todos:
Make sure to delete unnecessary points or to check all before merging:
 - [ ] Updated Changelog

-- 
# DO NOT MERGE UNTIL WK DOCS PR IS MERGED

### CODE REVIEW

File: docs/mkdocs.yml [hunk 1]
```diff
@@ -49,15 +49,18 @@ nav:
...
     - webknossos/data_formats.md
     - Keyboard Shortcuts: webknossos/keyboard_shortcuts.md
     - Backend REST API: webknossos/rest_api.md
-    - Code Readme: webknossos/code-readme.md
+    - Frontend Scripting API: https://webknossos.org/assets/docs/frontend-api/index.html" target="_blank
     - Changelog: webknossos/CHANGELOG.released.md
     - Migration Guide: webknossos/MIGRATIONS.released.md
-    - webknossos/publications.md
-    - webknossos/tooling.md
+    - [GitHub](https://github.com/scalableminds/webknossos)
```
Comment (hotzenklotz) [inline docs/mkdocs.yml:63]: I replace the "code readme" with a link to the GitHub repo. IMHO this more up-to-date and relevant

File: docs/mkdocs.yml [hunk 2]
```diff
@@ -49,15 +49,18 @@ nav:
     - webknossos/tasks.md
     - webknossos/sharing.md
     - webknossos/installation.md
+    - webknossos/automated_analysis.md
+    - webknossos/jobs.md
+    - webknossos/tooling.md
```
Comment (hotzenklotz) [inline docs/mkdocs.yml:54]: `Tooling` moved up from references. It felt kind of lost done there...

**Comment by hotzenklotz** [comment]
@normanrz Please review/approve.

**Review by normanrz — APPROVED** [review]

**Comment by jstriebel** [comment]
@hotzenklotz This PR broke CI also on master, I fixed it in #626

**Pull request merged**